aws/data_source_aws_eip.go

```diff
@@ -112,8 +120,30 @@ func dataSourceAwsEipRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("instance_id", eip.InstanceId)
 	d.Set("network_interface_id", eip.NetworkInterfaceId)
 	d.Set("network_interface_owner_id", eip.NetworkInterfaceOwnerId)
+
+	region := *conn.Config.Region
```

Comment:
Minor nit: We tend to access the provider region across the codebase via `region := meta.(*AWSClient).region` instead of via the client.